app/contributors/[slug]/page.tsx

```diff
@@ -264,6 +265,22 @@ export default async function Contributor({ params: { slug } }: Params) {
                 {contributor.highlights.eod_update}
               </dd>
             </div>
+            <div className="col-span-3 flex flex-col">
+              <dt className="order-2 mt-2 text-lg leading-6 font-medium text-primary-300">
+                Avg. PR Turnaround Time
+              </dt>
+              <dd className="order-1 text-5xl font-extrabold text-foreground truncate whitespace-nowrap">
+                {formatDuration(
+                  (contributor.activityData?.activity
+                    .map((act) => act.turnaround_time)
+                    .filter(Boolean)
+                    .reduce(
+                      (acc, curr, i, array) => acc! + curr! / array.length,
+                      0
+                    ) || 0) * 1000
```

Comment:
The averaging seems to be wrong here right?

Average shouldn't be dependent on the order of the elements right?

Your averaging technique is yielding different values when arr. is reversed.

Was weighted averaging intended?

<img width="682" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/2c9a878f-0154-4f8f-8207-b4fdd4e47595">

Reply:
forgot second param to the reduce

```js
arr.reduce((a, c, _, arr) => a + c / arr.length, 0)
```

<img width="498" alt="image" src="https://github.com/coronasafe/leaderboard/assets/29787772/a7e0c508-6244-43ed-9bd0-03c3815738d4">